src/models/CommonModel.ts

```diff
@@ -171,13 +269,15 @@ export class CommonModel extends CommonSchema<CommonModel> {
    * @param mergeFrom CommonModel to merge values from
    * @param originalSchema schema to use as original schema
    */
-  static mergeCommonModels(mergeTo: CommonModel | undefined, mergeFrom: CommonModel, originalSchema: Schema): CommonModel {
+  static mergeCommonModels(mergeTo: CommonModel | undefined, mergeFrom: CommonModel, originalSchema: Schema, alreadyIteratedModels: Map<CommonModel, CommonModel> = new Map()): CommonModel {
     if (mergeTo === undefined) return mergeFrom;
```

Comment:
I would consider opening `{}` even for simple if statements.

Reply:
@czlowiek488 can you clarify what you mean here? 🤔

Reply:
```
if (mergeTo === undefined) {
    return mergeFrom;
}
```

Reply:
Ahh, gonna keep it as is for now, added an issue to add eslint rule about this - https://github.com/asyncapi/generator-model-sdk/issues/190